.travis.yml

```diff
@@ -6,7 +6,7 @@ install:
   - mvn compile -DskipTests=true -Dmaven.javadoc.skip=true -Dcobertura.skip=true -B -V
 script: ./travis-script.sh
 after_success:
-  - test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_BRANCH == "master" && mvn site --settings ${HOME}/travis/settings.xml -B -V
+  - test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_BRANCH == "$SITE_BRANCH_NAME" && mvn site-deploy --settings ${HOME}/travis/settings.xml -B -V && mvn site:stage scm-publish:publish-scm --settings ${HOME}/travis/settings.xml -B -V
```

Comment:
Not sure what this `$SITE_BRANCH_NAME` variable is. I would expect to publish maven site only for master, as feature branches typically contain code that is in progress, so should probably not be published (we do not publish snapshots to Sonatype either). I also do not see where this variable is set.